naming/src/main/java/com/alibaba/nacos/naming/controllers/ServiceController.java

```diff
@@ -320,7 +320,7 @@ public String serviceStatus(HttpServletRequest request) throws Exception {
                         Loggers.SRV_LOG.debug("checksum of {} is not consistent, remote: {}, checksum: {}, local: {}",
                             serviceName, serverIp, checksum, service.getChecksum());
                     }
-                    serviceManager.addUpdatedService2Queue(checksums.namespaceId, serviceName, serverIp, checksum);
+                    serviceManager.addUpdatedServiceQueue(checksums.namespaceId, serviceName, serverIp, checksum);
```

Comment:
I think we can change `2` to `To`.If remove this word, the meaning of method will be changed.

Reply:
OK, I'll change it